### PR TITLE
[milvus] Fix missing clusterDomain in config template

### PR DESCRIPTION
**Pull request by chino540off**
## Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [ ] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)
- [x] PR only contains changes for one chart

### CODE REVIEW

File: charts/milvus/templates/config.tpl [hunk 1]
```diff
@@ -31,7 +31,7 @@ etcd:
 {{- else }}
   endpoints:
 {{- range $i := until ( .Values.etcd.replicaCount | int ) }}
-  - {{ $etcdReleaseName }}-{{ $i }}.{{ $etcdReleaseName }}-headless.{{ $namespace }}.svc.cluster.local:{{ $etcdPort }}
+  - {{ $etcdReleaseName }}-{{ $i }}.{{ $etcdReleaseName }}-headless.{{ $namespace }}.svc.{{ .Values.etcd.clusterDomain }}:{{ $etcdPort }}
```
Comment (haorenfsa) [inline charts/milvus/templates/config.tpl:34]: inside a `range` loop, Values should be referenced by  {{$.Values.xxx}}

ref: https://helm.sh/docs/chart_template_guide/variables/

**Review by haorenfsa — CHANGES_REQUESTED** [review]

File: charts/milvus/Chart.yaml [hunk 2]
```diff
@@ -1,9 +1,10 @@
+---
```
Comment (haorenfsa) [inline charts/milvus/Chart.yaml:1]: This change seems to be unnecessary

Comment (chino540off) [reply charts/milvus/Chart.yaml:1]: right, removed

File: charts/milvus/templates/config.tpl (hunk 1 above)
Comment (chino540off) [reply charts/milvus/templates/config.tpl]: fixed

**Pull request merged**